apps/src/componentLibrary/button/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.2.1](https://github.com/code-dot-org/code-dot-org/pull/56925)
```

Comment:
@levadadenys what's the version pattern here? Let me know if I need to change it!

Reply:
I feel like this is ok to make it 0.2.1, just update the link to be a link to this pr